supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -132,14 +132,22 @@ case class Run(runId: String,  jobDescriptor: BackendJobDescriptor, genomicsInte
     val currentStatus = status()
 
     if (!(previousStatus contains currentStatus)) {
-      // DONE: PBE: Moved state transition logging and "db" writes to our actor.
-      // Only thing "left" here via another ticket is:
-      // PBE deleted abort function registration
+      val prevStateName = previousStatus map { _.toString } getOrElse "-"
+      // TODO PBE: change println in this file to log statements
+      println(s"Status change from $prevStateName to $currentStatus")
```

Comment:
Remove this line. `JesAsyncBackendJobExecutionActor.poll` already prints out the line using a `log`.